tests/conftest.py

```diff
@@ -131,8 +132,9 @@ def es_version(client):
 @fixture
 def write_client(client):
     yield client
-    client.indices.delete(index="test-*", ignore=404)
-    client.indices.delete_template(name="test-template", ignore=404)
+    for index_name in client.indices.get(index="test-*", expand_wildcards="all"):
+        client.indices.delete(index=index_name)
```

Comment:
Deleting indices with a wildcard require a cluster setting in Elasticsearch 8. I had to specify `expand_wildcards` as some tests are closing indices.